tests/run_test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# tests/run_test_suite.sh.  Generated from run_test_suite.sh.in by configure.
+
+# run_test_suite
+#
+# Copyright (C) 2013,2016 by Swiss Library for the Blind, Visually Impaired and Print Disabled 
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the Lesser or Library GNU General Public License as
+# published by the Free Software Foundation; either version 3, or (at
+# your option) any later version.
+
+# This file is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the Library
+# GNU General Public License for more details.
+
+# You should have received a copy of the Library GNU General Public
+# License along with this program; see the file COPYING.  If not, write
+# to the Free Software Foundation, 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+
+# Create a temporary directory $tmp_dir in $TMPDIR (default /tmp).
+: ${TMPDIR=/tmp}
+{
+    tmp_dir=`(umask 077 && mktemp -d "$TMPDIR/fooXXXXXX") 2>/dev/null` && test -d "$tmp_dir"
+} || exit $?
+
+find-up () {
+    current=$(pwd)
+    path=$current
+    while [[ "$path" != "" && ! -e "$path/$1" ]]; do
+	path=${path%/*}
+    done
+    if [ "$path" != "" ]; then
+	echo $(realpath --relative-to=$current $path/$1)
```

Comment:
Does this work on MacOS?

Reply:
Or better: what is it supposed to do? Then I can try it.

Reply:
I hope it works on MacOS. It is supposed to walk up the directory tree and look for a given file. I got it from https://unix.stackexchange.com/a/22215

Reply:
I meant the `realpath` part. But I think I understand it now. And I tried it on Mac and it works. I'm not sure realpath is installed by default though.

Reply:
I'm using `dirname` now and dropped the use of `realpath`